src/data/blinkComponents.json

```diff
@@ -0,0 +1 @@
>DataUse","Internals>Network>FTP","Internals>Network>Filters","Internals>Network>HTTP2","Internals>Network>Library","Internals>Network>Logging","Internals>Network>NetInfo","Internals>Network>NetworkQuality","Internals>Network>Proxy","Internals>Network>QUIC","Internals>Network>SDCH","Internals>Network>SSL","Internals>OriginTrials","Internals>PageSecurityState","Internals>Permissions","Internals>Permissions>Model","Internals>Permissions>SiteEngagement","Internals>PlatformIntegration","Internals>Plugins","Internals>Plugins>Flash","Internals>Plugins>PDF","Internals>Plugins>Pepper","Internals>Preload","Internals>Printing","Internals>Printing>CUPS","Internals>Sandbox","Internals>Sandbox>SiteIsolation","Internals>Skia","Internals>SpeechSynthesis","Internals>Views","Internals>WebRTC","Manifest","Mobile","Mobile>WebAPKs","Mobile>WebView","Mobile>iOSWebView","OS>Systems>Bluetooth","Platform","Platform>Apps","Platform>Apps>AppLauncher>Install","Platform>Apps>BackgroundPage","Platform>Apps>BrowserTag","Platform>Apps>Feedback","Platform>Apps>FileManager","Platform>Apps>Shell","Platform>DevTools","Platform>DevTools>JavaScript","Platform>Extensions","Platform>Extensions>API","Platform>NaCl","Privacy","Security","Security>UX","Services>Chromoting","Services>CloudMessaging","Services>CloudPrint","Services>Invalidation","Services>SafeBrowsing","Services>Safebrowsing","Services>SignIn","Services>SupervisedUser","Services>Sync","Speed>Benchmark","Speed>Bisection","Speed>Dashboard","Speed>Telemetry","Speed>Tracing","Test","Test>Android","Test>Layout","Tools","Tools>Stability>Clusterfuzz","Tools>Stability>Libfuzzer","UI","UI>Accessibility","UI>Browser","UI>Browser>Autofill","UI>Browser>Bookmarks","UI>Browser>Bubbles","UI>Browser>Bubbles>PageInfo","UI>Browser>Core","UI>Browser>Downloads","UI>Browser>FirstRun","UI>Browser>FullScreen","UI>Browser>History","UI>Browser>Import","UI>Browser>Infobars","UI>Browser>Interstitials","UI>Browser>Mobile","UI>Browser>Mobile(Android)","UI>Browser>Mobile>CompositedUI","UI>Browser>Mobile>SearchWidget","UI>Browser>Mobile>Share","UI>Browser>NewTabPage","UI>Browser>Offline","UI>Browser>Omnibox","UI>Browser>Passwords","UI>Browser>Passwords>Generation","UI>Browser>Payments","UI>Browser>Permissions>Prompts","UI>Browser>Preferences","UI>Browser>Preferences>Protector","UI>Browser>Previews","UI>Browser>PrintPreview","UI>Browser>Profiles","UI>Browser>Search","UI>Browser>Search>ContextualSearch","UI>Browser>Spellcheck","UI>Browser>TabCapture","UI>Browser>TabContents","UI>Browser>TabStrip","UI>Browser>Themes","UI>Browser>Toolbar","UI>Browser>VR","UI>Browser>WebUI","UI>Browser>Zoom","UI>GFX","UI>Input","UI>Input>Text>IME","UI>Input>VirtualKeyboard","UI>Localization","UI>Notifications","UI>OSIntegration","UI>ProximityAuth","UI>Security>UrlFormatting","UI>Settings","UI>Shell","UI>Shell>GestureNav","UI>Shell>LockScreen","UI>Shell>Shelf","UI>Shell>StatusArea","UI>Shell>TouchView","UI>Shell>Wallpaper","UI>Shell>WindowManager","UI>TaskManager","Webstore"]
```

Comment:
We should also have a way to keep this file updated. @jpmedley what about adding a task to gulp to update it an overwrite the file. It comes sorted and formatted from the endpoint :)